tests/test_raster_index_util.py

```diff
@@ -1,9 +1,78 @@
 import shutil
+
+import numpy as np
 import pytest
+import rasterio
+from osgeo import gdal
 
 from cropclassification.util import raster_index_util, raster_util
 from tests.test_helper import SampleData
 
+gdal.UseExceptions()
+
+
+def create_gdal_raster(
+    fname,
+    values,
+    *,
+    gt=None,
+    gdal_type=None,
+    nodata=None,
+    scale=None,
+    offset=None,
+    bands=None,
```

Comment:
```suggestion
    band_descriptions=None,
```
Is probably a clearer name?